azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -191,6 +191,7 @@ public int toFlags() {
     SCHEDULE(0x0000008),
     METRICS(0x0000010),
     CREATEPROJECTS(0x40000000), // Only used for roles
+    UPLOADPROJECTS(0x0008000),
```

Comment:
Adding this permission makes sense only if we are willing to do the work to take advantage of the finer grained control. 

Do you plan to change dr elephant user from admin role to upload and create project roles?